src/traces/bar/style_defaults.js

```diff
@@ -23,6 +23,12 @@ module.exports = function handleStyleDefaults(traceIn, traceOut, coerce, default
 
     coerce('marker.line.width');
     coerce('marker.opacity');
+    var pattern = coerce('marker.pattern.shape');
```

Comment:
Let's call this variable `patternShape`.

Reply:
Fixed in 51614783013bdc48ea9d8076f44a31a8da95b5a5